cli/args/mod.rs

```diff
@@ -1188,6 +1199,27 @@ fn resolve_local_node_modules_folder(
   Ok(Some(canonicalize_path_maybe_not_exists(&path)?))
 }
 
+fn resolve_deno_modules_folder(
+  cwd: &Path,
+  flags: &Flags,
+  maybe_config_file: Option<&ConfigFile>,
+) -> Option<PathBuf> {
+  let use_deno_modules_dir = flags
+    .deno_modules_dir
+    .or_else(|| maybe_config_file.and_then(|c| c.deno_modules_dir()))
+    .unwrap_or(false);
+  if !use_deno_modules_dir {
+    None
+  } else if let Some(config_path) = maybe_config_file
+    .as_ref()
+    .and_then(|c| c.specifier.to_file_path().ok())
+  {
+    Some(config_path.parent().unwrap().join("deno_modules"))
+  } else {
+    Some(cwd.join("deno_modules"))
```

Comment:
Why not canonicalize like we do for `node_modules`?

Reply:
There's no need. For npm it's necessary though because the resolved modules are canonicalized within the directory. I can add a comment here explaining this.